content/en/docs/ops/ambient/getting-started/gtwapi_test.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154,SC2034
+
+# Copyright 2023 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GATEWAY_API="true"
+
+# @setup profile=none
+source "content/en/docs/ops/ambient/getting-started/test.sh"
+
+# @cleanup
```

Comment:
You need to add the full set of cleanup instructions for the gateway API test here, or nothing will be cleaned up in the premature exit (error) case.